src/exercise/03-extra.html

```diff
@@ -1,32 +1,16 @@
-<!-- Using JSX -->
-<!-- http://localhost:3000/isolated/exercise/03.html -->
-
 <body>
   <div id="root"><div className="container">Hello World!</div></div>
   <script src="https://unpkg.com/react@18.1.0/umd/react.development.js"></script>
   <script src="https://unpkg.com/react-dom@18.1.0/umd/react-dom.development.js"></script>
 
   <script src="https://unpkg.com/@babel/standalone@7.12.4/babel.js"></script>
 
-  <!-- 🐨 add Babel to the page.
-       💰 Here is the script tag that'll do the job:
-  -->
-
-  <script type="text-babel">
-    // 🐨 on the script tag above, change `type="module"`
-    // to `type="text/babel"` so babel will compile this code for the browser to run.
+  <script type="text/babel">
+    const className = 'container'
+    const children = 'Hello World!!!'
 
-    // 🐨 re-implement this using JSX!
-    <!-- const element = React.createElement('div', {
-      className: 'container',
-      children: 'Hello World',
-    }) -->
+    const element = <div className>{children}</div>
```

Comment:
shorthand operation does not work with react. In this case you'll need to do:
`const element = <div className={className}>{children}</div>`

Reply:
Ohh interesting!